Author: Rasmus Thomsen <oss@cogitri.dev>
Description: LC_ADDRESS is a GNU specific extension, so musl doesn't offer this. Since
"US" is the default, let's just use that for now instead.
Upstream: https://gitlab.gnome.org/GNOME/evolution-data-server/-/issues/236
--- a/src/addressbook/libebook-contacts/e-phone-number-private.cpp.orig	2020-07-17 20:43:05.948364253 +0200
+++ b/src/addressbook/libebook-contacts/e-phone-number-private.cpp	2020-07-17 20:42:30.988385824 +0200
@@ -119,7 +119,11 @@
 	 * From outside this is a C library, so we better consult the
          * C infrastructure instead of std::locale, which might divert. */
 	if (region_code == NULL || region_code[0] == '\0')
+#ifdef LC_ADDRESS
 		return _e_phone_number_cxx_region_code_from_locale (setlocale (LC_ADDRESS, NULL));
+#else
+		return "US";
+#endif
 
 	return region_code;
 }
